### DISCUSSION TITLE
✘ No compatible model found for 'en_core_wb_sm' (spaCy v2.3.7).

### DISCUSSION THREAD

**Discussion by SanjuEpic** [explosion/spaCy#11535] [Help: Other Questions] [answered]
## How to reproduce the behaviour
When i tried to download the model using the below code, i received this weird error statement.
```
python -m spacy download en_core_wb_sm
```
## Result I get

```
✘ No compatible model found for 'en_core_wb_sm' (spaCy v2.3.7).
```

## Your Environment

* Operating System: Linux
* Python Version Used: 3.8.10
* spaCy Version Used: 2.3.7

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @SanjuEpic , I'll convert this into a discussion since it's not an issue. You have misspelt the model name. It's `en_core_web_sm`. Note that it's `web`, not `wb`.

**Reply by SanjuEpic** [reply]
Thanks for helping me out, my bad 😔
